app/code/Magento/Checkout/Test/Mftf/Test/StorefrontVisiblePasswordFieldForUnregisteredEmailOnCheckoutTest.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontVisiblePasswordFieldForUnregisteredEmailOnCheckoutTest">
+        <annotations>
+            <features value="Checkout"/>
+            <stories value="Visible password field for unregistered e-mail on Checkout"/>
+            <title value="Visibility password field for unregistered e-mail on Checkout process"/>
+            <description value="Guest should not be able to see password field if entered unregistered email"/>
+            <stories value="Guest Checkout"/>
+            <testCaseId value="MC-14695"/>
```

Comment:
I don't think that there is a specific `testCaseId`. So it's better to remove it.